Makefile

```diff
@@ -137,17 +137,34 @@ main.exe: $(OBJS) $(NXDK_DIR)/lib/xboxkrnl/libxboxkrnl.lib
 	@echo "[ AS       ] $@"
 	$(VE) $(AS) $(NXDK_ASFLAGS) $(ASFLAGS) -c -o '$@' '$<'
 
-%.inl: %.vs.cg $(VP20COMPILER)
+%.vs.asm: %.vs.cg $(VP20COMPILER)
 	@echo "[ CG       ] $@"
-	$(VE) $(CGC) -profile vp20 -o $@.$$$$ $< $(QUIET) && \
-	$(VP20COMPILER) $@.$$$$ > $@ && \
-	rm -rf $@.$$$$
-
-%.inl: %.ps.cg $(FP20COMPILER)
+	$(VE) $(CGC) -profile vp20 -o $@.tmp $< $(QUIET); \
+	head -n 1 $@.tmp > $@; \
```

Comment:
Untested wether this is still necessary. I added this because I got "bad header" errors. But then I realized that my comment used `//` instead of `#`; so this hack might not be necessary.